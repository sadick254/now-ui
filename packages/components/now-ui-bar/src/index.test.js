import {h} from 'preact';
import {render} from '@testing-library/preact';
import Bar from '.';

describe('Bar', () => {
  it('should render correctly', () => {
    const {container} = render(
      <Bar
        width="48px"
        height="150px"
        backgroundColor="#8C4C44"
        fillColor="#D2634C"
        fillHeight="20%"
        titleAbove="27.5%"
        titleBelow="18-24"
        barBorderRadius="4px"
        fillBorderRadius="0 0 4px 4px"
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
