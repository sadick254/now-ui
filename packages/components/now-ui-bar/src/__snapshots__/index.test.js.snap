// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar should render correctly 1`] = `
<div>
  <div
    class="iZzsrl "
    height="150px"
    width="48px"
  >
    27.5%
    <div
      backgroundcolor="#8C4C44"
      borderradius="4px"
      class="ljApbG "
    >
      <div
        borderradius="0 0 4px 4px"
        class="cgiiXG "
        fillcolor="#D2634C"
        fillheight="20%"
      />
    </div>
    18-24
  </div>
</div>
`;
